Gemfile

```diff
@@ -32,7 +32,7 @@ gem 'require_all'
 # background processing and workflow
 gem 'sidekiq'
 gem 'sidekiq-unique-jobs'
-gem 'sidekiq-scheduler', '~> 2.0'
+gem 'sidekiq-scheduler', '2.0.19' # Latest version 2.1.7 uses rufus-scheduler a bit differently -- 8/3/17
```

Comment:
this just broke a single test because of the way the changes return a rufus timestamp. Probably easy to fix the spec to accommodate, but for now bumped version back.